internal/app/wwctl/overlay/edit/root.go

```diff
@@ -22,11 +22,13 @@ var (
 	}
 	CreateDirs bool
 	PermMode   int32
+	Command    string
 )
 
 func init() {
 	baseCmd.PersistentFlags().BoolVarP(&CreateDirs, "parents", "p", false, "Create any necessary parent directories")
 	baseCmd.PersistentFlags().Int32VarP(&PermMode, "mode", "m", 0755, "Permission mode for directory")
+	baseCmd.PersistentFlags().StringVarP(&Command, "command", "c", "", "Command to run editor")
```

Comment:
What is the use case for `--command` vs just using `env EDITOR="emacs -nw" wwctl [...]`? I looked at `git commit`, for example, and it doesn't seem to have such an argument.

Reply:
The use case is to use `sed` `yq` directly on files.  This way you don't have to know the parent directory for the overlay and wwctl will do the "right thing" when copying from a distribution to a local overlay.

In OpenHPC we use it for things like this (this is new and needed for the 3.3 release)
```
wwctl overlay cat host /etc/hosts.ww | \
  sed -e 's_\({{$node.Id}}{{end}}\)_{{$node.Id}}.localdomain \1_g' | \
  EDITOR=tee wwctl overlay edit host /etc/hosts.ww
```

With this it would look like
```
wwctl overlay edit --command "sed -e 's_\({{$node.Id}}{{end}}\)_{{$node.Id}}.localdomain \1_g'"  host /etc/hosts.ww
```